webapp-src/src/Login/NoPasswordForm.js

```diff
@@ -45,7 +47,16 @@ class NoPasswordForm extends Component {
     }
   }
 
+  gotoManageUsers(e) {
+    e.preventDefault();
+    document.location.href = this.state.config.LoginUrl + "?callback_url=" + encodeURIComponent([location.protocol, '//', location.host, location.pathname].join('')) + "&scope=" + encodeURIComponent(this.state.config.profile_scope) + "&prompt=select_account";  
```

Comment:
Reloading the page looses all the context with parameters `scopes`, `redirect_uri` and other query parameters, I changed this code to use this code instead:

```javascript
messageDispatcher.sendMessage('App', {type: 'SelectAccount'});
```

Reply:
Ouch—very true. I forgot we are in the Login context here. 

Thanks for merging, @babelouest.